test/soapui/Dolibarr-soapui-project.xml

```diff
@@ -3637,7 +3637,7 @@
         <xsd:all>
           <xsd:element name="type" type="xsd:string"/>
           <xsd:element name="priority" type="xsd:string"/>
-          <xsd:element name="connexion" type="xsd:string"/>
```

Comment:
This is a field name choosed in the protocol of this xml file so we should not modify it.

Reply:
4 eyes are better than 2 ;-) .  I scrutenized the changes several times to revert those, but I missed this one.